### DISCUSSION TITLE
prefix topic on ZB Bridge

### DISCUSSION THREAD

**Discussion by Siggemada** [arendst/Tasmota#10289] [Q&A] [answered]
I changed the full topic to Stockholm/ZBBridge but I am still getting:

`14:24:08.031 MQT: tele/Plug2/SENSOR = {"ZbReceived":{"Plug2":{"Device":"0x5595","Name":"Plug2","EnergyTotal":"0x000000000019","Endpoint":1,"LinkQuality":147}}}
14:26:15.003 MQT: tele/Plug1/SENSOR = {"ZbReceived":{"Plug1":{"Device":"0x3383","Name":"Plug1","EnergyTotal":"0x000000000009","Endpoint":1,"LinkQuality":152}}}
14:27:21.414 MQT: Stockholm/ZBBridge/STATE = {"Time":"2020-12-28T14:27:21","Uptime":"0T00:15:09","UptimeSec":909,"Vcc":3.514,"Heap":26,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"xxx","BSSId":"yyy","Channel":6,"RSSI":100,"Signal":-30,"LinkCount":1,"Downtime":"0T00:00:03"}}
14:29:06.958 MQT: tele/Plug2/SENSOR = {"ZbReceived":{"Plug2":{"Device":"0x5595","Name":"Plug2","EnergyTotal":"0x000000000019","Endpoint":1,"LinkQuality":149}}}`

Is it supposed to do this?

Thanks

**Comment by Siggemada** [comment, marked as answer]
Is there a syntax for the full topic setting that I missed? I filled in "Stockholm/ZBBridge".

**Reply by sfromis** [reply]
FullTopic should always contain the %prefix% variable (for tele/stat/cmnd). Normally, it works best to let FullTopic be the default (or swap the order if you like), and then set Topic according to your setup.

**Reply by Siggemada** [reply]
Ok, I have a wan over vpn for two sites. I would like to prefix the sites by locale. But is that not possible?

Or would "%prefix%/Stockholm/ZBBridge" be a valid full topic?

**Reply by Siggemada** [reply]
Ok, seems like full topic "%prefix%/Stockholm/%topic%/" works.

**Reply by sfromis** [reply]
What I mentioned was including %prefix% in the full topic always, but it does not need to be in the beginning. My starting point for what you describe might be %topic%/%prefix% and then topic Stockholm/ZBBridge
